views/admin-page-header.php

```diff
@@ -15,7 +15,20 @@
 // phpcs:ignore WordPress.Security.NonceVerification.Recommended
 $progress_planner_active_frequency = isset( $_GET['frequency'] ) ? \sanitize_text_field( \wp_unslash( $_GET['frequency'] ) ) : 'monthly';
 
+do_action( 'progress_planner_admin_page_header_before' );
 ?>
+<?php if ( isset( $_GET['content-scan-finished'] ) ) : // phpcs:ignore WordPress.Security ?>
+	<div class="prpl-widget-wrapper prpl-top-notice">
+	<div class="inner-content">
+		<h1><?php esc_html_e( "We've successfully scanned your site's content.", 'progress-planner' ); ?></h1>
+		<p><?php esc_html_e( 'Would you like a tour of the plugin now?', 'progress-planner' ); ?></p>
+			<div id="prpl-onboarding-submit-grid-wrapper">
+				<button class="prpl-button-primary" onclick="prplStartTour()"><?php esc_html_e( 'Yes, let\'s go!', 'progress-planner' ); ?></button>
+			</div>
+		</div>
+	</div>
```

Comment:
Should this notice also have a `Skip tour` button?

Reply:
Also, when the tour finishes, we should hide the notice

Reply:
Did the latter, will add a close button.

Reply:
Added close buttons, to the Playground notice too.